### PR TITLE
Validate only submitted Project Euler solution

### PR DESCRIPTION
**Pull request by dhruvmanila**
Messing around with GitHub Actions, trying to see if this is possible:
We will run checks only on the submitted Project Euler solution and not all.

### **Describe your change:**

* [x] Improve Project Euler validate solutions script
* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by dhruvmanila** [comment]
Is this useful? @cclauss 

I was messing around with GitHub Actions and wrote some functions so that the validate_solutions script only checks for the submitted solution and if there are none, it will default to check all solutions. Also, I have scheduled this to be run every day for all solutions in case we break something.

File: scripts/validate_solutions.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 import pathlib
 from types import ModuleType
 from typing import Dict, List
 
 import pytest
 
+try:
+    import github
+except ImportError:
+    pass
```
Comment (cclauss) [inline scripts/validate_solutions.py:14]:
```suggestion
    github = None
```
This allows easy to understand `if github:` logic.

**Review by cclauss — APPROVED** [review]
`Draft` usually means _not yet ready for review_.

File: scripts/validate_solutions.py (hunk 1 above)
Comment (dhruvmanila) [reply scripts/validate_solutions.py:14]: I will be using `requests` to directly make the API call as there is only one.

**Comment by dhruvmanila** [comment]
Note to me: Configure the bot to ignore the `scripts/` directory.

**Comment by dhruvmanila** [comment]
> Draft usually means not yet ready for review.

Sorry for that, I just wanted you to a second opinion on whether this was worth doing.

**Pull request merged**